crates/store/re_format_arrow/src/lib.rs

```diff
@@ -96,15 +96,13 @@ impl std::fmt::Display for DisplayMetadata {
             &metadata
                 .iter()
                 .map(|(key, value)| {
-                    format!(
-                        "{prefix}{}: {}",
-                        if *trim_keys { trim_key(key) } else { key },
-                        if *trim_values {
-                            trim_name(value)
-                        } else {
-                            value
-                        },
-                    )
+                    let key = if *trim_keys { trim_key(key) } else { key };
+                    let value = if *trim_values {
+                        trim_name(value)
+                    } else {
+                        value
+                    };
+                    format!("{prefix}{key}: {value}",)
```

Comment:
Style: Instead of using `let` bindings, you could also use named parameters to the `format! macro:



```rs
format!("{prefix}{key}: {value}",
    key = …,
    value = …,
)
```